src/client/pages/ThemaPagina/ThemaPagina.tsx

```diff
@@ -48,6 +51,7 @@ export default function ThemaPagina({
   icon = <ThemaIcon />,
   pageContentTop,
   linkListItems = [],
+  buttonItems = [],
```

Comment:
Ik zou ipv buttonItems een prop toevoegen voor arbitraire content onder de LinkList. Dat is net wat flexibeler. 

iets generieks als `pageContentTopSecondary` ofzo.